css-brochure-form/styles.css

```diff
@@ -0,0 +1,69 @@
+main {
+  width: 400px;
+  margin: 0 auto;
+}
+
+h2 {
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  text-transform: uppercase;
+  font-size: small;
+  color: #ec5d58;
+  font-weight: 425;
+  letter-spacing: 0.05em;
+  margin-block-end: 0;
+}
+
+fieldset {
+  border: none;
+}
+
+input,
+select {
+  margin: 5px;
+  padding: 0.5em;
+  width: 27em;
+  height: 2.75em;
+  border: 1px;
+  border-color: rgb(214 211 211);
+  border-style: solid;
+  border-radius: 5px;
+  font-family: AtlasGrotesk, sans-serif;
+}
+
+input {
+  margin-block-start: 1px;
+}
+
+label {
+  padding-left: 1.5em;
+  text-align: left;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  text-transform: uppercase;
+  font-size: x-small;
+  font-weight: 500;
+}
+
+button {
+  background-color: #ec5d58;
+  color: white;
+  border: none;
+  border-radius: 18px;
+  padding: 0.4em;
+  width: 5.6em;
+  text-align: center;
```

Comment:
I would use text align on the parent of the button so you can use it to move it to the right of the div. That would eliminate the need for a lot of margin.

Reply:
<img width="480" alt="Screen Shot 2022-10-13 at 10 12 10 AM" src="https://user-images.githubusercontent.com/113091744/195662359-4142d5f4-8478-4bf3-beaf-c2ac57c77155.png">